otherlibs/systhreads/st_stubs.c

```diff
@@ -219,7 +219,7 @@ static caml_thread_t caml_thread_new_info(void)
   th->exit_buf = NULL;
   th->backtrace_pos = 0;
   th->backtrace_buffer = NULL;
-  th->backtrace_last_exn = caml_create_root(Val_unit);
+  th->backtrace_last_exn = Val_unit;
```

Comment:
I don't see where `backtrace_last_exn` is now registered with the GC; surely we need to register it as a generational root and then use the correct modification and delete functions?